bootstrap_views_generator.gemspec

```diff
@@ -36,7 +36,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_runtime_dependency 'railties', '>= 4.0', '<= 7'
+  spec.add_runtime_dependency 'railties', '>= 7'
```

Comment:
Same suggest as on your other PR https://github.com/tarellel/bootstrap_views_generator/pull/2#discussion_r1390025884

Can you please make this `>= 6` or just remove the upper rails version limit entirely please.